.gitignore

```diff
@@ -5,3 +5,4 @@
 /dist
 *.swp
 .idea
+steve
```

Comment:
Why is this being ignored? Is this a directory with some generated/built artifacts?

Reply:
It's the build artifact when building locally.